htdocs/fourn/class/fournisseur.commande.class.php

```diff
@@ -2403,14 +2403,21 @@ public function getDispachedLines($status = -1)
 		$ret = array();
 
 		// List of already dispatched lines
-		$sql = "SELECT p.ref, p.label,";
+		$sql = "SELECT p.rowid as pid, p.ref, p.label,";
 		$sql .= " e.rowid as warehouse_id, e.ref as entrepot,";
-		$sql .= " cfd.rowid as dispatchedlineid, cfd.fk_product, cfd.qty, cfd.eatby, cfd.sellby, cfd.batch, cfd.comment, cfd.status, cfd.fk_commandefourndet";
-		$sql .= " FROM ".MAIN_DB_PREFIX."product as p,";
-		$sql .= " ".MAIN_DB_PREFIX."commande_fournisseur_dispatch as cfd";
+		$sql .= " cfd.rowid as dispatchlineid, cfd.fk_product, cfd.qty, cfd.eatby, cfd.sellby, cfd.batch, cfd.comment, cfd.status, cfd.datec";
+		$sql .= " ,cd.rowid, cd.subprice";
```

Comment:
This 2 new fields seems to not be used.

Reply:
Same answer as for the first point: due to a specific development.